agents/unix/main.c

```diff
@@ -1542,6 +1542,15 @@ rcf_ch_rpc_server_thread(void *ready, int argc, char *argv[])
 }
 #endif
 
+#if TE_CONSTRUCTOR_AVAILABLE
+TE_CONSTRUCTOR_PRIORITY(101)
+#endif
+static void
+log_init(void)
```

Comment:
It looks unused if !TE_CONSTRUCTOR_AVAILABLE. Shouldn't #endif be moved below after the function?